config/routes.rb

```diff
@@ -1,7 +1,10 @@
 Rails.application.routes.draw do
-  root to: "home#index"
+  root to: "submissions#index"
 
   devise_for :users, controllers: { registrations: 'registrations' }
-
   resources :users, only: :show
+
+  resources :submissions do
+    post 'create_comment'
```

Comment:
In the past, I've seen create actions just as expressed `post '<resource>'`, like `post 'comment'`. I want to say this follows REST more closely, but I don't know if there's an existing Rails convention.

What do you think about removing the `create_` here?

Reply:
I think that's a great idea, @nhunzaker, but I can also see a couple other options here:

1. Modify `submissions#create` to handle both posts **and** comments.
1. Create a new `comments#create` action to handle comment submissions.